ctlcmd/motioncmd.py

```diff
@@ -222,13 +224,17 @@ def run(self, args):
     lumi = []
     unc = []
     total = len(args.x)
-
+    
+    n = 0
+    test1 = []
+    test2 = []
+    self.openroot("halign")
     ## Running over mesh.
     for xval, yval in self.start_pbar(zip(args.x, args.y)):
       self.check_handle()
       self.move_gantry(xval, yval, args.scanz)
       lumival, uncval = self.readout(args, average=True)
-      self.write_standard_line((lumival, uncval), det_id=args.detid)
+      self.fillroot([lumival,uncval],det_id=args.detid)
```

Comment:
If possible, run the python formatter `yapf` [1] on the python files, so that we cat a consistent file format.

[1] https://github.com/google/yapf